test/prop_restc_body.erl

```diff
@@ -0,0 +1,22 @@
+-module(prop_restc_body).
```

Comment:
Nice stuff, when you have time it be nice if I could have an intro to how these tests work. I have basic understanding of PBT but never used it in practice. And I don't think we have PBT's anywhere else in the codebase currently. In can read up on Prop3r as well.